src/assets/usage-examples-code/sprites/draw_sprite/draw_sprite-2-draw-sprite.cs

```diff
@@ -0,0 +1,37 @@
+/*
+  Usage Example: drawing a sprite
+*/
+
+using System;
+using SplashKitSDK;
+
```

Comment:
Your code looks great! To make it even more streamlined, you can add "using static SplashKitSDK.SplashKit;" at the beginning of your file. This will allow you to remove the SplashKit. prefix from your method calls, such as OpenWindow, making the code cleaner and easier to read.

Reply:
Done